python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -1166,3 +1172,15 @@ def print_other_sessions(self):
         if len(other_sessions) > 0:
             logger.warning("%i other concurrent sessions:\n * %s" % (len(other_sessions), "\n * ".join(other_sessions)))
 
+    def has_loaded(self, obj):
+
+        try:
+            index = self.find(obj)
+        except:
+            return False
```

Comment:
This exception handler should be more precise. I'm guessing it should be `ObjectNotInRegistryError`. Also, the function could do with a doc string.